ambry-api/src/main/java/com.github.ambry/store/StoreKey.java

```diff
@@ -37,6 +37,18 @@
    */
   public abstract String getID();
 
+  /**
+   * Get accountId of StoreKey, return -1 if unknown.
+   * @return accountId
+   */
+  public abstract short getAccountId();
```

Comment:
Could you add `@overrides` to the corresponding methods in the `BlobId` class?

Reply:
yes.